index.js

```diff
@@ -5,6 +5,7 @@ const controlAccess = require('control-access');
 const token = process.env.GITHUB_TOKEN;
 const username = process.env.GITHUB_USERNAME;
 const origin = process.env.ACCESS_ALLOW_ORIGIN;
+const maxRepos = process.env.MAX_REPOS ? process.env.MAX_REPOS : 6;
```

Comment:
I think `process.env.MAX_REPOS` needs to be wrapped in a `Number`. :thinking:

Reply:
Yes it needs to be converted to `number`. Also the use of ternary operator is a bit redundant here. I would use

```js
const maxRepos = Number(process.env.MAX_REPOS) || 6
```